### PR TITLE
Feat/no tweets on wall [Delivers #106147038]

### PR DESCRIPTION
**Pull request by chiefGui**
@fosterdill @harby @ozeias @ThadeuBrito @petrbela

### CODE REVIEW

**Comment by ThadeuBrito** [comment]
:+1:

File: src/Menu.jsx [hunk 1]
```diff
@@ -1,64 +1,68 @@
...
+      if (this.state.isOptionsVisible) {
+        return (
+          <ul className="components-options__list">
+            { this.renderAllOptions() }
+          </ul>
+        )
+      }
+
+      return null
+    }.bind(this)
```
Comment (petrbela) [inline src/Menu.jsx]: `() =>` already binds to this

Comment (petrbela) [inline src/Menu.jsx]: actually, you don't need to make this a function at all

Comment (petrbela) [inline src/Menu.jsx]: if you do want to make it a function for readability, put it on the class

File: src/components/live-displays/Grid/Grid.jsx [hunk 2]
```diff
@@ -1,35 +1,37 @@
...
+  }
 
   render() {
-    this.prepareAssets();
+    let assets = this.props.assets
+
+    if (this.props.twitter) {
+      assets[0].tweetOnly   = true
+      assets[5].withCaption = true
+      assets = assets.slice(0, 8)
```
Comment (petrbela) [inline src/components/live-displays/Grid/Grid.jsx]: can we move this to some demo config file?

File: src/components/live-displays/Grid/GridAsset.jsx [hunk 3]
```diff
@@ -1,15 +1,47 @@
...
+      }
+
+      if (this.props.asset.tweetOnly) {
+        return this.renderTweetOnly()
+      }
+
+      return this.renderAsset()
+    }
+
+    return asset()
```
Comment (petrbela) [inline src/components/live-displays/Grid/GridAsset.jsx]: the function is just making it more complicated imho

File: src/components/live-displays/Grid/Grid.jsx (hunk 2 above)
Comment (chiefGui) [inline src/components/live-displays/Grid/Grid.jsx]: It's not about a demo data. Instead, through that fragment of code I am saying that on position 0 and 5 always will have a tweetOnly and withCaption asset.

**Comment by petrbela** [comment]
:+1: with comments

**Pull request merged**